clickhouse/tests/docker/init.sql

```diff
@@ -1,3 +1,3 @@
-CREATE TABLE tableau (id Int64) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{layer}-{shard}/tableau', '{replica}') PARTITION BY id ORDER BY id;
-CREATE TABLE tableau_distributed as tableau ENGINE = Distributed(cluster_1, default, tableau, rand());
+CREATE TABLE IF EXISTS tableau (id Int64) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{layer}-{shard}/tableau', '{replica}') PARTITION BY id ORDER BY id;
```

Comment:
IF NOT EXISTS

Reply:
![image](https://user-images.githubusercontent.com/1266346/183028351-988cab9d-298d-42e4-bc56-d6df4735a5aa.png)